serdect/src/slice.rs

```diff
@@ -147,7 +150,32 @@ where
 
         deserializer.deserialize_str(StrVisitor)
     } else {
-        Vec::deserialize(deserializer)
+        struct VecVisitor;
+
+        impl<'de> Visitor<'de> for VecVisitor {
+            type Value = Vec<u8>;
+
+            fn expecting(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
+                write!(formatter, "a bytestring")
+            }
+
+            fn visit_bytes<E>(self, v: &[u8]) -> Result<Self::Value, E>
+            where
+                E: Error,
+            {
+                Ok(v.into())
+            }
+
+            #[cfg(feature = "alloc")]
```

Comment:
```suggestion
```
The function this is in is already gated by `feature = "alloc"`.

Reply:
Fixed.